src/test/java/org/opensearch/neuralsearch/processor/NormalizationProcessorIT.java

```diff
@@ -94,7 +94,15 @@ public void testResultProcessor_whenOneShardAndQueryMatches_thenSuccessful() {
         createSearchPipelineWithResultsPostProcessor(SEARCH_PIPELINE);
         String modelId = getDeployedModelId();
 
-        NeuralQueryBuilder neuralQueryBuilder = new NeuralQueryBuilder(TEST_KNN_VECTOR_FIELD_NAME_1, "", modelId, 5, null, null);
+        NeuralQueryBuilder neuralQueryBuilder = new NeuralQueryBuilder(
```

Comment:
Do we need to change this in this PR?

Reply:
yes, we're changing a constructor by adding `image` field. Not directly related to functionality, but it will fail otherwise.